File: cpp/src/Neuron.cpp
#include "../include/Neuron.h"
#include <iostream>

using namespace std;

Neuron::Neuron() : _value(0)
{
}

Neuron::~Neuron()
{
}

double Neuron::getValue()
{
    return _value;
}

void Neuron::setValue(double value)
{
    _value = value;
}

File: ats/src/main/java/fr/automated/trading/systems/pricingmodel/pricingstrategies/neuralnetworks/neuron/Neuron.java
package fr.automated.trading.systems.pricingmodel.pricingstrategies.neuralnetworks.neuron;

public abstract class Neuron implements INeuron {

	private double value;

	public double getValue() {
		return value;
	}

	public void setValue(double value) {
		this.value = value;
	}
	
	public Neuron() {
	}

}
